app/assets/javascripts/application.js.erb

```diff
@@ -389,11 +389,13 @@ var Lobsters = new _Lobsters();
 
 $(document).ready(function() {
   var $olcomments = $("ol.comments");
-  $olcomments.on("click", ".comment a.downvoter", function() {
+  $olcomments.on("click", ".comment .voters > a + div + a", function() {
+    console.log("downvote");
```

Comment:
Looks like you left this in by accident, as well as another `console.log("upvote");` a bit further down.

Reply:
silly me, will get removed asap